### ISSUE TITLE
do_link_login_flow of login.py fails due to missing config parameter to _login_client

### ISSUE THREAD

**Issue by mikedarcy** [globus/globus-identifiers-client#1]
Exception in function `do_link_login_flow` in line 67 of `login.py` when calling `_login_client` due to missing `config` parameter.

**Comment by mikedarcy** [comment]
`TypeError: _login_client() takes exactly 1 argument (0 given)`

Sorry I indeed should have included the exception but I thought it would be obvious once you looked at the code in the call chain.

Here is what the user submitted (via email):
```
[ec2-user@ip-172-31-36-59 globus-identifiers-client]$ /home/ec2-user/.local/share/virtualenvs/globus-identifiers-client-nwQ1wExI/bin/globus-identifiers-client login --force
Traceback (most recent call last):
File "/home/ec2-user/.local/share/virtualenvs/globus-identifiers-client-nwQ1wExI/bin/globus-identifiers-client", line 11, in <module>
load_entry_point('globus-identifiers-client==0.1.0', 'console_scripts', 'globus-identifiers-client')()
File "/home/ec2-user/.local/share/virtualenvs/globus-identifiers-client-nwQ1wExI/lib/python2.7/site-packages/globus_identifiers_client-0.1.0-py2.7.egg/identifiers_client/main.py", line 308, in main
File "/home/ec2-user/.local/share/virtualenvs/globus-identifiers-client-nwQ1wExI/lib/python2.7/site-packages/globus_identifiers_client-0.1.0-py2.7.egg/identifiers_client/main.py", line 54, in login
File "/home/ec2-user/.local/share/virtualenvs/globus-identifiers-client-nwQ1wExI/lib/python2.7/site-packages/globus_identifiers_client-0.1.0-py2.7.egg/identifiers_client/login.py", line 67, in do_link_login_flow
TypeError: _login_client() takes exactly 1 argument (0 given)
```

**Comment by madduri** [comment]
I am running into the same bug.

**Comment by NickolausDS** [comment]
This should be fixed by #3 in version 0.1.0, and I think this issue can now be closed.
